From 07e35a1ae99b7a95b6893e10239c420650e7c187 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Mon, 22 Aug 2022 20:36:47 +0200
Subject: [PATCH] Fix condition for Breeze style

This was supposed to be Windows, not Android

(cherry picked from commit 7c6a197791cd71163753677281c28345c727fab3)
---
 src/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.cpp b/src/main.cpp
index ff118808..6d33a3a4 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -81,7 +81,7 @@ int main(int argc, char *argv[])
 
     QApplication app(argc, argv);
 
-#if defined Q_OS_ANDROID || defined Q_OS_MAC
+#if defined Q_OS_WIN || defined Q_OS_MAC
     QApplication::setStyle(QStringLiteral("breeze"));
 #endif
 
-- 
2.37.2

